hera_mc/tests/test_sys_handling.py

```diff
@@ -33,6 +33,12 @@ def test_publish_summary(self):
         msg = self.h.publish_summary()
         self.assertEqual(msg, 'Not on "main"')
 
+    def test_other_hookup(self):
+        H = cm_hookup.Hookup(at_date='now', session=self.test_session)
+        H.reset_cache(None)
+        self.assertEqual(H.cached_hookup_dict, None)
+        H.show_hookup_cache_file_info()
```

Comment:
Can we devise a higher-level test here? The assertion right now is basically testing that we can successfully set a variable to None, which is not super informative.

Also, the call to `show_hookup_cache_file_info` seems a bit out of place here. I think that should be its own test case.